tests/functional/spec/section_summary.spec.js

```diff
@@ -31,11 +32,11 @@ describe('Section Summary', function() {
     it('When I continue on the section summary page, Then I should be taken to the next section', function() {
       return browser
         .click(PropertyDetailsSummaryPage.submit())
-        .getUrl().should.eventually.contain(FinalSummaryPage.pageName);
+        .getUrl().should.eventually.contain(HouseType.pageName);
     });
   });
 
-  describe('Given I start a Test Section Summary survey and complete to Final Summary', function() {
+  describe.skip('Given I start a Test Section Summary survey and complete to Final Summary', function() {
```

Comment:
intentional skip? if so, why?

Reply:
Not intentional, I re-enabled it and it highlighted a change in behaviour. 

I've amended the tests to reflect the new behaviour - the difference to previous behaviour is when clicking edit from a final summary and then submit on the answer, the user is directed to the section summary (if the section is complete), rather than back to the final summary.